### PR TITLE
Proposal of doc's modifications 

### PR DESCRIPTION
**Pull request by HubLot**
Hi, 

You'll find some modifications of the current documentation hosted in RTD. Here a link to those modifications : https://buildh.readthedocs.io/en/modif_doc/ 
I didn't touch the content (beside some typos) but more the overall organization of the index and the menu :
  - I populated the index.rst file to reflect a proper "home" page of the project with the content found in the buildh.md file. I realize having a beautiful image could be a great asset.
  - I split the buildh.md file into installation.md and usage.md (so it can be reflected in the left menu).
  - I regroup the pages for the algorithms, notebooks/tutorials and API into sub-menus.
 
I still need to validate the notebooks (i spot bad links).

Let me know what you think!

### CODE REVIEW

**Comment by pierrepo** [comment]
I like it very much :star_struck:  I find it easier to browse.
 But I let @patrickfuchs get the last word on this matter.

File: docs/notebooks/Notebook_01_buildH_calc_OP.ipynb [hunk 1]
```diff
@@ -403,7 +403,7 @@
     "\n",
     "$$OP_{stem}(CH) = \\frac{ \\frac{1}{N} \\sum_{i=1}^{N} OP(CH)_i }{\\sqrt{N}}$$\n",
     "\n",
-    "where *N* is the number of lipids, $OP(CH)_i$ is the mean order parameter of a given CH of residue $i$ (this $OP(CH)_i$ is itself an average of the order parameter of a given CH of residue *i* over frames). In fact, we first average over frames, then we average over residues to compute the STEM.\n",
+    "where *N* is the number of lipids, $OP(CH)_i$ is the mean order parameter of a given CH of residue $i$ (this $OP(CH)_i$ is itself an average of the order parameter of a given CH of residue *i* over frames). In fact, we first average over frames, then we average over residues to compute the STEM. More information about the order parameter can be found in the documentation: [Order parameters and statistics](../order_parameter.md)\n",
```
Comment (patrickfuchs) [inline docs/notebooks/Notebook_01_buildH_calc_OP.ipynb:406]: Thinking about that, I realize I don't use the same indices in this formula and the one in "OP & statistics". I need to change this!

**Review by patrickfuchs — APPROVED** [review]

**Comment by patrickfuchs** [comment]
Thanks very much @HubLot for this great PR, this organization is definitely better and indeed easier to browse 😄 ! The only thing I'd like to change is the name "Additionnal features" which I don't like at all... If you have suggestions, please let me know. If not, I can merge the PR and we can think about it later.
Last I definitely agree we need a nice image. I have an idea. That would be nice if we could put on top of the menu like for example in [pyemma](http://www.emma-project.org/latest/). @pierrepo , do you know if it is doable with MyST?

**Comment by pierrepo** [comment]
> That would be nice if we could put on top of the menu like for example in pyemma.

Yes, it *should* be possible: https://jupyterbook.org/customize/config.html

File: docs/notebooks/Notebook_01_buildH_calc_OP.ipynb (hunk 1 above)
Comment (HubLot) [reply docs/notebooks/Notebook_01_buildH_calc_OP.ipynb:406]: Okay, I let you correct this once this is merged

**Comment by HubLot** [comment]
> 
> Thanks very much @HubLot for this great PR, this organization is definitely better and indeed easier to browse 😄 ! The only thing I'd like to change is the name "Additionnal features" which I don't like at all... If you have suggestions, please let me know. If not, I can merge the PR and we can think about it later.

Thanks! I aggree with the "additionnal features", it's not the best chosen term. Maybe "Advanced usage" or "Advanced practice" or "advanced topic" ? something around that.

I'm gonna add also a quick paragraph about the tests

**Comment by patrickfuchs** [comment]
> Thanks! I aggree with the "additionnal features", it's not the best chosen term. Maybe "Advanced usage" or "Advanced practice" or "advanced topic" ? something around that.

Thanks! Well, I'm not sure about the word *advanced* since we're not describing advanced things, e.g. the supported lipids is not something advanced. I thought about "Additionnal details", but it's not very different 😝 Maybe "Good to know" or "Good to know before running **buildH**"?

> I'm gonna add also a quick paragraph about the tests

OK ! Do you plan to add this paragraph within this PR?

**Comment by HubLot** [comment]
> Thanks! Well, I'm not sure about the word _advanced_ since we're not describing advanced things, e.g. the supported lipids is not something advanced. I thought about "Additionnal details", but it's not very different 😝 Maybe "Good to know" or "Good to know before running **buildH**"?

True, I was more focus on the "mixture lipid" and "PBC" which we can considered *advanced*. I like "Additional details"

**Comment by patrickfuchs** [comment]
> True, I was more focus on the "mixture lipid" and "PBC" which we can considered _advanced_. I like "Additional details"

OK let's go for "Additional details"! Could you please modify this, and then I merge?

**Comment by HubLot** [comment]
I'll write a small notebook for buildh as a library

**Comment by patrickfuchs** [comment]
> I'll write a small notebook for buildh as a library

Great, thanks! On my side I started the mini paper, if you add deal with Notebook04 I'll be faster to finish the mini paper 😉 After that, only adding the cholesterol will be left, and I have it close to ready. Almost there 💪 !

**Comment by patrickfuchs** [comment]
So ready for the merge of this PR?

**Comment by HubLot** [comment]
Notebook added, now it's ready to merge :)

**Comment by patrickfuchs** [comment]
Yeaaah, you're super fast 😎 !!!

**Pull request merged**